lib/components/TextArea.tsx

```diff
@@ -59,6 +62,15 @@ export const TextArea = forwardRef(
     const textareaRef = useRef<HTMLTextAreaElement>(null);
     const [scrolledAmount, setScrolledAmount] = useState(0);
 
+    function getMarkupString(
```

Comment:
this is a pure function and should be moved to the global scope

https://oxc.rs/docs/guide/usage/linter/rules/unicorn/consistent-function-scoping